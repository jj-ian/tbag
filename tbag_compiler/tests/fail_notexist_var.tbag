/* Authors: Maria */
true {
	testvar();
	endgame;
}

func int testvar() {
        i = 7;
	return 0;
}
