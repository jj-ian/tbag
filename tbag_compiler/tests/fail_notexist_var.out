Fatal error: exception Failure("undeclared identifier i")
